manifests/init.pp

```diff
@@ -166,16 +166,11 @@
   }
 
   if $pulp or $reverse_proxy_real {
-    class { 'certs::apache':
-      hostname => $foreman_proxy_fqdn,
-      require  => Class['certs'],
-    }
-    ~> class { 'foreman_proxy_content::reverse_proxy':
```

Comment:
Does moving this still trigger refreshes if the certificates change? e.g. restarting Apache

Reply:
I think this should be sufficient:
https://github.com/theforeman/puppet-foreman_proxy_content/blob/a622bd3fb2f5a8eb6b1881d1828a51dd0f40800c/manifests/reverse_proxy.pp#L30